edpop-record-ontology.ttl

```diff
@@ -212,3 +228,41 @@ edpoprec:physicalDescription a rdf:property ;
                              rdfs:domain edpoprec:BibliographicalRecord ;
                              rdfs:range edpoprec:Field .
 
+### FIELDS SPECIFIC FOR BIOGRAPHICAL RECORDS
+
+edpoprec:name a rdf:property ;
+              skos:prefLabel "Name"@en ;
+              rdfs:domain edpoprec:BiographicalRecord ;
+              rdfs:range edpoprec:Field .
+
+edpoprec:variantName a rdf:property ;
+              skos:prefLabel "Name"@en ;
+              rdfs:domain edpoprec:BiographicalRecord ;
+              rdfs:range edpoprec:Field .
+
+edpoprec:placeOfBirth a rdf:property ;
+              skos:prefLabel "Place of birth"@en ;
+              rdfs:domain edpoprec:BiographicalRecord ;
+              rdfs:range edpoprec:LocationField .
+
+edpoprec:placeOfDeath a rdf:property ;
+              skos:prefLabel "Place of death"@en ;
+              rdfs:domain edpoprec:BiographicalRecord ;
+              rdfs:range edpoprec:LocationField .
+
+edpoprec:placeOfActivity a rdf:property ;
+              skos:prefLabel "Place of activity"@en ;
+              rdfs:domain edpoprec:BiographicalRecord ;
+              rdfs:range edpoprec:LocationField .
+
+edpoprec:timespan a rdf:property ;
+                  skos:prefLabel "Timespan"@en ;
+                  skos:description "The years that the person was alive or the entity was existing" ;
+                  rdfs:domain edpoprec:BiographicalRecord ;
+                  rdfs:range edpoprec:DatingField .
+          
+edpoprec:activity a rdf:property ;
+                  skos:prefLabel "Activity"@en ;
+                  skos:description "An activity that the entity is known for" ;
+                  rdfs:domain edpoprec:BiographicalRecord ;
+                  rdfs:range edpoprec:Field .
```

Comment:
I might expect something like

```ttl
edpoprec:activity rdfs:domain edpoprec:BiographicalRecord ; rdfs:range edpoprec:Activity .
edpoprec:placeOfActivity rdfs:domain edpoprec:Activity .
edpoprec:timespan rdfs:domain edpoprec:Activity .
```

That is, that `placeOfActivity` and `timespan` have activities as their domain.

If a biographical record describes a person with multiple activities, and `placeOfActivity` is described for both, this setup would be ambiguous.

Reply:
Nested fields make sense to me. It's kind of the strength of RDF that you can easily create data structures with any level of depth.